dlang.org.ddoc

```diff
@@ -237,16 +237,15 @@ $(SUBMENU_MANUAL
     $(SUBMENU_LINK $(ROOT_DIR)spec/spec.html, Language Reference)
     $(SUBMENU_LINK $(ROOT_DIR)phobos/index.html, Library Reference)
     $(SUBMENU_LINK $(ROOT_DIR)library/index.html, Library Reference Preview)
-    $(SUBMENU_LINK $(ROOT_DIR)dmd-windows.html, Command-line Reference)
+    $(SUBMENU_LINK $(ROOT_DIR)dmd.html, Command-line Reference)
     $(SUBMENU_LINK_DIVIDER $(ROOT_DIR)comparison.html, Feature Overview)
     $(SUBMENU_LINK $(ROOT_DIR)articles.html, Articles)
 )
 NAVIGATION_RESOURCES=
 $(SUBMENU_MANUAL
     $(SUBMENU_LINK https://wiki.dlang.org/Books, Books)
     $(SUBMENU_LINK https://wiki.dlang.org/Tutorials, Tutorials)
-    $(SUBMENU_LINK_DIVIDER $(ROOT_DIR)tools.html, D-Specific Tools)
-    $(SUBMENU_LINK https://wiki.dlang.org/Editors, Editors)
+    $(SUBMENU_LINK_DIVIDER https://wiki.dlang.org/Editors, Editors)
```

Comment:
Hmm after having had a look at the wiki and https://github.com/zhaopuming/awesome-d, I am not so sure how about to date the wiki is.